Vagrantfile

```diff
@@ -56,16 +76,30 @@ Vagrant.configure(2) do |config|
     end
   end
 
-  # WINDOWS WORKER (win server 2019)
+  ############ Windows worker node #1 ############
   config.vm.define :winw1 do |winw1|
-    winw1.vm.host_name = "winw1"
-    winw1.vm.box = "sig-windows-dev-tools/windows-2019"
+    winw1.vm.box = "mloskot/sig-windows-dev-tools-windows-2019"
```

Comment:
in passing quick note this doesnt work w/ QEMU but the other box *does* work with QEMU.  any thoughts on why?

Reply:
No idea really, but perhaps QEMU trips over the pre-install VirtualBox Guest Additions?

I assume those boxes are used temporarily as enablers to move forward with SWDT testing and requirements discovery.
It would be good if QEMU folks tried to figure out how to build bespoke boxes.
Then, we can see how to combine building of boxes for both providers into common pipeline.